cnxdb/migrations/20200506144128_update_trees_slug_values.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+
+import os
+import time
+from datetime import timedelta
+from functools import wraps
+
+import requests
+from dbmigrator import deferred
+from dbmigrator import logger
+from cnxcommon.urlslug import generate_slug
+
+
+ARCHIVE_DOMAIN = os.getenv('ARCHIVE_DOMAIN')
+VARNISH_HOSTS = os.getenv('VARNISH_HOSTS', '').split(',')
+VARNISH_PORT = int(os.getenv('VARNISH_PORT', 80))
+
+
+BATCH_SIZE = 1000
+TREE_QUERY = """\
+WITH RECURSIVE t(node, title, path, value, depth, corder, is_collated) AS (
```

Comment:
As this is now it will recompute all slugs. Do we want to refine / scope in any way?